verification/verify_test.go

```diff
@@ -86,17 +86,29 @@ func TestVerify(t *testing.T) {
 	assert.Equal(ErrPixelMismatch, err)
 
 	// Check pixel count succeeds w/o external verifier
-	pixels, err := pixels("../server/test.flv")
+	pxls, err := pixels("../server/test.flv")
 	assert.Nil(err)
 	data = &net.TranscodeData{Segments: []*net.TranscodedSegmentData{
-		{Url: "../server/test.flv", Pixels: pixels},
-		{Url: "../server/test.flv", Pixels: pixels},
+		{Url: "../server/test.flv", Pixels: pxls},
+		{Url: "../server/test.flv", Pixels: pxls},
 	}}
 	renditions := [][]byte{{123}, {234}}
-	res, err = sv.Verify(&Params{Results: data, Orchestrator: &net.OrchestratorInfo{}, Renditions: renditions})
+	res, err = sv.Verify(&Params{Results: data, Orchestrator: &net.OrchestratorInfo{TicketParams: &net.TicketParams{}}, Renditions: renditions})
 	assert.Nil(err)
 	assert.NotNil(res)
 
+	// Check pixel count fails when len(params.Results.Segments) != len(params.Renditions)
```

Comment:
We should have tests that check that sig verification is actually invoked. Structuring those tests is a bit tricky right now since we can't stub the `VerifySig()` implementation. Options include:

1. Generating some sample data and using it in the tests. Ex. Create 2 byte slices, compute the keccak256 hash of each, compute the keccak256 hash of the concatenation of the 2 hashes, sign this hash with an account and then use the signature + byte slices + account in this test
2. Configure the `Policy` with a `sigVerifier` interface that defines the `VerifySig` method and instantiate the `Policy` with an implementation that calls `crypto.VerifySig()`. See the [pm package](https://github.com/livepeer/go-livepeer/blob/75aa225b4667c57644f3289b760f656651e8407a/pm/sigverifier.go#L23) for an example.

Reply:
hopefully this will suffice?  f60f3881ac9dc8f8a80a651ad88dc9eba156b740